queenbee/job/job.py

```diff
@@ -10,6 +10,8 @@ class Job(BaseModel):
 
     A Job is an object to submit a list of arguments to execute a Queenbee recipe.
     """
+    api_version: constr(regex='^Job$') = 'Job'
```

Comment:
This should probably be 'v1beta1'. What do you think about using an enumeration for api_versions?

```suggestion
    api_version: constr(regex='^v1beta1$') = 'v1beta1'
```

Reply:
Thanks for the fix! So my understanding was that Enumerations are not allowed because of C#? I would consider this item to be the same as the `type` key. It's a fixed value for this object and helps discern it from other objects.

Reply:
Enums are fine in C#. It was just we couldn't use some specific keys.

I'm thinking that in long-term we want to be able to support more than one. We can add the enum at that point. If we do now it will only have a single value.